flink-connectors/flink-connector-elasticsearch2/src/main/resources/META-INF/NOTICE

```diff
@@ -1,133 +1,85 @@
- Copyright (c) 2014 Michael Barker
- Copyright (c) 2014 Matt Warren
- All rights reserved.
-
- Redistribution and use in source and binary forms, with or without
- modification, are permitted provided that the following conditions are met:
-
- 1. Redistributions of source code must retain the above copyright notice,
-    this list of conditions and the following disclaimer.
-
- 2. Redistributions in binary form must reproduce the above copyright notice,
-    this list of conditions and the following disclaimer in the documentation
-    and/or other materials provided with the distribution.
-
- THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
- AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
- IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
- ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
- LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
- CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
- SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
- INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
- CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
- ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
- THE POSSIBILITY OF SUCH DAMAGE.
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- com.carrotsearch:hppc:0.7.1
+- com.fasterxml.jackson.core:jackson-core:2.6.6
+- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.6.6
+- com.fasterxml.jackson.dataformat:jackson-dataformat-smile:2.6.6
+- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.6.6
+- com.google.guava:guava:18.0
+- com.google.code.findbugs:jsr305:1.3.9
+- com.ning:compress-lzf:1.0.2
+- com.spatial4j:spatial4j:0.5
+- com.tdunning:t-digest:3.0
+- com.twitter:chill-java:0.7.4
+- commons-cli:commons-cli:1.3.1
+- io.netty:netty:3.10.5.Final
+- joda-time:joda-time:2.5
+- org.apache.lucene:lucene-analyzers-common:5.5.0
+- org.apache.lucene:lucene-core:5.5.0
+- org.apache.lucene:lucene-backward-codecs:5.5.0
+- org.apache.lucene:lucene-grouping:5.5.0
+- org.apache.lucene:lucene-highlighter:5.5.0
+- org.apache.lucene:lucene-join:5.5.0
+- org.apache.lucene:lucene-memory:5.5.0
+- org.apache.lucene:lucene-misc:5.5.0
+- org.apache.lucene:lucene-queries:5.5.0
+- org.apache.lucene:lucene-queryparser:5.5.0
+- org.apache.lucene:lucene-sandbox:5.5.0
+- org.apache.lucene:lucene-spatial:5.5.0
+- org.apache.lucene:lucene-spatial3d:5.5.0
+- org.apache.lucene:lucene-suggest:5.5.0
+- org.elasticsearch:elasticsearch:2.3.5
+- org.elasticsearch:securesm:1.0
+- org.joda:joda-convert:1.7
+- org.yaml:snakeyaml:1.15
+
+This project bundles the following dependencies under the BSD License.
+See bundled licenses for details.
+
+- org.hdrhistogram:HdrHistogram:2.1.6
```

Comment:
it is licensed under both: https://github.com/HdrHistogram/HdrHistogram/blob/6afbfd18eca6efefacd4bb429bf6adc6c66f6489/LICENSE.txt

> For users of this code who wish to consume it under the "BSD" license
rather than under the public domain or CC0 contribution text mentioned
above, the code found under this directory is *also* provided under the
following license (commonly referred to as the BSD 2-Clause License). This
license does not detract from the above stated release of the code into
the public domain, and simply represents an additional license granted by
the Author.

Reply:
I'll change the license to CC0, based on the presto filesystem.